src/encryption/api.go

```diff
@@ -0,0 +1,75 @@
+package encryption
+
+import (
+	"context"
+	"fmt"
+	"keboola-as-code/src/client"
+	"strings"
+
+	"github.com/go-resty/resty/v2"
+	"go.uber.org/zap"
+)
+
+type EncryptionApi struct {
+	apiHost string
+	client  *client.Client
+	logger  *zap.SugaredLogger
+}
+
+// Error represents Encryption API error structure
+type Error struct {
+	Message     string `json:"error"`
+	ErrCode     int    `json:"code"`
+	ExceptionId string `json:"exceptionId"`
+}
```

Comment:
musel som tu zaviest vlastnu strukturu pre error pretoze nesedi stym co vracia Storage API. Konkretne code je tu int a u storage api je to string.
Priklad chyby:
```
{"error":"The componentId parameter is required.","code":0,"exceptionId":"exception-11f4677967634fe292a8b95287c95082","status":"error"}     
```

Reply:
:+1: